### ISSUE TITLE
Request/bug [afr.com.au]

### ISSUE THREAD

**Issue by chanpreetsal** [bpc-clone/bypass-paywalls-chrome-clean#20]
Describe issue/request: AFR.COM doesn't work anymore. Only displays the first couple of paragraphs which are free. It does sometime randomly work though 

Link of paywalled article: https://www.afr.com/chanticleer/coles-has-a-bit-of-a-booze-problem-20240430-p5fnrb

Actions you tried yourself and if it works (clear cookies, disabe Javascript, change user-agent/referer, reader view, archive sites and/or add a custom site):  cleared cookies. turned on the second option for Australian news corp "no googlebot" but that didn't work either. tested on ios with Adguard - same issue. Tested on google chorme - same issue 

Browser (main version):  Edge 124.xx

BPC-extension version (number): v3.6.6.2

PS always first search (domain) name in (all) issues (& use latest ext. version).
Also request host permission for unlisted domain (by clearing cookies once by BPC-icon; X = no fix).

**Comment by LukePrior** [comment]
+1

The contents of the article are still accessible within the HTML so it should be possible to bypass, see also: https://www.reddit.com/r/AusFinance/comments/1cgo49s/did_the_afr_just_get_a_real_paywall/

**Comment by chanpreetsal** [comment]
I checked others owned by nine entertainment and they seemed fine. 

Australian Financial Review, a newspaper and subscription news website. ... - Not working

Tested the below one they working fine
Brisbane Times, 
The Age, a newspaper
The Sydney Morning Herald,
WAtoday,

**Comment by LordGMLP** [comment]
> Indeed, initial plain text of article in page source (not updated to latest version though). I'll add a fix later.
> 
> Site is part of Australia Nine Entertainment (not News Corp): also new paywall for other sites of group?

Tested on other sites owned by the group. Like chanpreetsal said, all working EXCEPT AFR.com. It's Nine's premium newspaper with a more expensive subscription.

**Comment by LordGMLP** [comment]
> Site fixed in latest master (or next release).

Tested and working. Thank you very much. Please flag as Resolved.

**Comment by speedboattoyota** [comment]
Hi I think the afr made another change to the website and its not working again. Can you please take a look? This is after the fix that you pushed through.

**Comment by LordGMLP** [comment]
> Hi I think the afr made another change to the website and its not working again. Can you please take a look? This is after the fix that you pushed through.

Working fine for me. Make sure you download the Zip file and manually install on Chrome or Firefox. The .crx file for Chrome and .xpi file for Firefox has not been updated yet. Make sure the latest version says 3.6.6.4

**Comment by LukePrior** [comment]
Yeah not working any more for me: https://www.afr.com/companies/transport/bonza-directors-say-they-were-blindsided-by-lease-cancellation-20240502-p5folc

**Comment by LordGMLP** [comment]
> Yeah not working any more for me: https://www.afr.com/companies/transport/bonza-directors-say-they-were-blindsided-by-lease-cancellation-20240502-p5folc

I can see the whole article. I monitor the AFR constantly for work. What is the version of your plugin?

**Comment by LukePrior** [comment]
> What is the version of your plugin?

v3.6.6.4

Hmmm weird because I updated to latest master yesterday and it was working but only stopped in past hour or so.

**Comment by blueys45** [comment]
+1 also just encountered the same issue on Afr site as well

**Comment by LordGMLP** [comment]
> > What is the version of your plugin?
> 
> v3.6.6.4
> 
> Hmmm weird because I updated to latest master yesterday and it was working but only stopped in past hour or so.

Now that you've mentioned it, what is the last paragraph that you can see with the plugin active? I can see up to _"Aviation industry sources said Bonza had been in talks with financiers and aircraft suppliers that appeared to have stalled."_

**Comment by LukePrior** [comment]
> Now that you've mentioned it, what is the last paragraph that you can see with the plugin active?

With the plugin enabled I see nothing
![image](https://github.com/bpc-clone/bypass-paywalls-chrome-clean/assets/22492406/135539c0-7eff-4f80-9681-a679ad2d21f1)

**Comment by LordGMLP** [comment]
Definitely weird. Here's the whole article for me: 
![image](https://github.com/bpc-clone/bypass-paywalls-chrome-clean/assets/161445082/1c916dba-adb6-403d-938c-2324a31af35f)

**Comment by blueys45** [comment]
Yeap just tried as well and I get the same with plugin enabled. Haven't tried without it

**Comment by LukePrior** [comment]
> Definitely weird. 

Hmmm if you try clearing cache I wonder if it will still work, AFR could be doing a staged deployment of changes so only some users are affected?

**Comment by blueys45** [comment]
> > Definitely weird.
> 
> Hmmm if you try clearing cache I wonder if it will still work, AFR could be doing a staged deployment of changes so only some users are affected?

Nope doesn't work for me either way

**Comment by LordGMLP** [comment]
Both of you are right. I just tested on another computer. Deleted all caches and cookies (Chrome version). No longer working @bpc-clone. All it does is a blank page like blueys45 said.

**Comment by dhdco** [comment]
afr.com no longer works.
Tried to update to v3.6.6.5 and cleared cache + cookies, still shows a blank page.

Been using this wonderful addon for ages, just wanna pop in and say thanks!

**Comment by LordGMLP** [comment]
> Site fixed (again) in latest master v3.6.6.6 (paywall script now also removes full article from page source/dom).

Tested and working as expected. Much appreciate the quick fix. Btw older version 3.6.6.4 is still working on Chrome _without_ cookies/cache cleared, but I'm not taking the chance it will work correctly for much longer.

**Comment by blueys45** [comment]
Thank you! Have installed 3.6.6.6 and can confirm it's working now on afr

**Comment by henry-white89** [comment]
@bpc-clone legends like you make the world go around. I was having the same problem as others in this thread, updated to 3.6.6.6 and now it's working perfectly.

Thank you

**Comment by chanpreetsal** [comment]
I am finding that some articles on AFR not fully load. I am on version 3.6.7.0

For example, the below two articles dont load
https://www.afr.com/technology/did-russia-just-sabotage-our-samsung-watch-ultra-20240801-p5jycr 
https://www.afr.com/property/residential/here-are-the-10-most-expensive-homes-in-australia-20240730-p5jxlj

whereas these load fine
https://www.afr.com/world/north-america/australians-to-get-fast-entry-to-us-in-global-program-20240806-p5jzto
https://www.afr.com/politics/federal/unis-to-be-capped-at-40pc-overseas-students-20240805-p5jzm0

Thanks
